dataprofiler/labelers/base_data_labeler.py

```diff
@@ -9,15 +10,14 @@
 
 import numpy as np
 import pandas as pd
-import pkg_resources
 
 from dataprofiler._typing import DataArray
 
 from .. import data_readers
 from . import data_processing
 from .base_model import BaseModel
 
-default_labeler_dir = pkg_resources.resource_filename("resources", "labelers")
+default_labeler_dir = str(importlib.resources.files("resources")) + "/labelers"
```

Comment:
does this need a future for 3.8 or a conditional on py version?

Similar issue: https://github.com/mu-editor/pup/issues/35

Reply:
Instead of `str` around + "labelers",

i suggest following the method described in: https://github.com/capitalone/DataProfiler/issues/855

Was there an issue with using the `joinpath`? << should avoid any pathing join issues as well.

Reply:
thank you for following up! 
The suggested `default_labeler_dir = importlib.resources.files("resources").joinpath("labelers")` 
causes this error to appear: `dataprofiler/labelers/data_processing.py:148: error: No overload variant of "join" matches argument types "Traversable", "str"`

When I implemented this: `default_labeler_dir = str(importlib.resources.files("resources")) + "/labelers"`
All `mypy` errors disappeared on my local machine. This is probably due to my Python version being greater than 3.8, or would it be something else?